src/v/cluster/topic_table.h

```diff
@@ -75,7 +75,7 @@ namespace cluster {
  *                                        force_cancel
  */
 
-class topic_table {
+class topic_table : public ss::enable_lw_shared_from_this<topic_table> {
```

Comment:
where is `shared_from_this()` used?

Reply:
oops, I was trying out a few things and forgot to undo this..